vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1612,7 +1612,17 @@ EXPORT(bool, sceGxmProgramIsDiscardUsed, const SceGxmProgram *program) {
 EXPORT(bool, sceGxmProgramIsEquivalent, const SceGxmProgram *programA, const SceGxmProgram *programB) {
     assert(programA);
     assert(programB);
-    return UNIMPLEMENTED();
+
+    if (!programA)
+        LOG_ERROR("ProgramA is invalid");
+    if (!programB)
+        LOG_ERROR("ProgramB is invalid");
+
+    STUBBED("Compare programs, not output");
+    if (programA == programB)
+        return true;
+    else
+        return false;
```

Comment:
```suggestion
    if (!programA || !programB)
        return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);

    return programA == programB ? true : false;
```

Reply:
remove also both assert

Reply:
function returns bool, RET_ERROR would return nonsense there, isn't better to just log the error and return false?